Makefile

```diff
@@ -2262,21 +2271,19 @@ git$X: git.o GIT-LDFLAGS $(BUILTIN_OBJS) $(GITLIBS)
 		$(filter %.o,$^) $(LIBS)
 
 help.sp help.s help.o: command-list.h
-hook.sp hook.s hook.o: hook-list.h
+builtin/bugreport.sp builtin/bugreport.s builtin/bugreport.o: hook-list.h
```

Comment:
I'm not sure what these hook vs bugreport changes are doing here.

Reply:
This was intentionally pulled in from an upstream commit because of a build error that was (for some reason) only manifesting in `make dist` on MacOS 11: https://github.com/vdye/git_osx_installer/runs/4387194585?check_suite_focus=true

I included it "early" (rather than letting it naturally show up as part of next release's rebase) because the error caused by incorrect dependencies is blocking to any mid-cycle releases we may want to do. It does have some things not explicitly needed (e.g., the removal of `hook-list.h` as a dependency of `help`), but including the full commit seemed clearer.